src/main/java/io/ebean/enhance/common/EnhanceContext.java

```diff
@@ -65,7 +65,7 @@ public EnhanceContext(ClassBytesReader classBytesReader, String agentArgs, Agent
     this.filterEntityTransactional = new FilterEntityTransactional(manifest);
     this.filterQueryBean = new FilterQueryBean(manifest);
 
-    this.detectQueryBean = Distill.convert(manifest.getEntityPackages());
+    this.detectQueryBean = Distill.convert(manifest.getQuerybeanPackages());
```

Comment:
This was wrong IMHO

Reply:
Nope, getEntityPackages() is correct.  The enhancement of query beans is really 2 things ... enhancing the query beans themselves (which are under the entity packages) and the potential enhancement of callers.